__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+
+describe("GET /api/topics", () => {
+  it("responds with an array with 'slug' and 'description' properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topics = res.body.topics;
+        expect(Array.isArray(topics)).toEqual(true);
+        expect(topics.length).toBeGreaterThan(0);
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
```

Comment:
This is a good test! You could drop line 12, checking that the data is an array, because you are using a `forEach`